docs/api-reference.md

```diff
@@ -59,6 +59,33 @@ If `children` contains more than one child, `oneChild` function will throw an er
 
 If `children` is `nil` or contains no children, `oneChild` will return `nil`.
 
+### Roact.createBinding
+```
+Roact.createBinding(initialValue) -> Binding, updateFunction
+where
+	updateFunction: (newValue) -> ()
+```
+
+The first value returned is a `Binding` object, which will typically be passed as a prop to a Roact host component. The second is a function that can be called with a new value to update the binding.
+
+A `Binding` has the following API:
+
+#### getValue
+```
+Binding:getValue() -> value
+```
+
+Returns the internal value. This isn't typically necessary, but can be helpful when updating a binding relative to its current value.
```

Comment:
We should note that using `getValue()` in render is probably a bad idea (for now) since your component won't re-render.